install.sh

```diff
@@ -144,11 +144,11 @@ _install_rust_and_cargo() {
     *) echo "Please answer yes or no." ;;
     esac
   done
-  echo "rustup installed! Installing with cargo"
-  _install_cargo
+  echo "rustup installed!"
```

Comment:
The prints from rustup installer makes it very clear that rust was installed, so this line is superfluous now.

Reply:
[d2401e6](https://github.com/shuttle-hq/shuttle/pull/1610/commits/d2401e6dcbfdf598eeb4d328fb2d080be4692091)